doc/man3/OSSL_PARAM.pod

```diff
@@ -33,76 +33,75 @@ Arrays of this type can be used for the following purposes:
 
 =item * Setting parameters for some object
 
-The caller sets up the C<OSSL_PARAM> array and calls some function
+The caller sets up the B<OSSL_PARAM> array and calls some function
 (the I<setter>) that has intimate knowledge about the object that can
-take the data from the C<OSSL_PARAM> array and assign them in a
+take the data from the B<OSSL_PARAM> array and assign them in a
 suitable form for the internal structure of the object.
 
 =item * Request parameters of some object
 
-The caller (the I<requestor>) sets up the C<OSSL_PARAM> array and
+The caller (the I<requestor>) sets up the B<OSSL_PARAM> array and
 calls some function (the I<responder>) that has intimate knowledge
 about the object, which can take the internal data of the object and
 copy (possibly convert) that to the memory prepared by the
-I<requestor> and pointed at with the C<OSSL_PARAM> C<data>.
+I<requestor> and pointed at with the B<OSSL_PARAM> I<data>.
 
 =item * Request parameter descriptors
 
-The caller gets an array of constant C<OSSL_PARAM>, which describe
+The caller gets an array of constant B<OSSL_PARAM>, which describe
 available parameters and some of their properties; name, data type and
 expected data size.
 For a detailed description of each field for this use, see the field
 descriptions below.
 
 The caller may then use the information from this descriptor array to
-build up its own C<OSSL_PARAM> array to pass down to a I<setter> or
+build up its own B<OSSL_PARAM> array to pass down to a I<setter> or
 I<responder>.
 
 =back
 
-=head2 C<OSSL_PARAM> fields
+Normally, the order of the an B<OSSL_PARAM> array is not relevant.
```

Comment:
Have all of these cases been resolved?  They were legacy only as far as I am aware.

Might it be better to omit this and leave the processing order completely undefined?  It could save future pain if we do.

Reply:
Depends...  there was the argument that this would make it easier on streaming protocols, and the KDF to providers effort still has this support in the provider implementations, and it is being used in the TLS code, so maybe that needs some extra work...